packages/web/src/utils/config.spec.ts

```diff
@@ -295,6 +295,15 @@ describe('getBaseWebpackPartial', () => {
         )
       ).toBeTruthy();
     });
+
+    it('should tolerate being empty', () => {
```

Comment:
I think it's better to be more restrictive on your expectations as on the previous tests, ie not copy-webpack-plugin present.